apps/prairielearn/src/ee/lib/ai-grading.ts

```diff
@@ -108,57 +112,44 @@ async function generateGPTPrompt({
   // Examples
   for (const example of example_submissions) {
     if (rubric_items.length > 0 && example.manual_rubric_grading_id) {
+      // Note that the example may have been graded with a different rubric,
+      // or the rubric may have changed significantly since the example was graded.
+      // We'll show whatever items were selected anyways, since it'll likely
+      // still be useful context to the LLM.
       const rubric_grading_items = await queryRows(
         sql.select_rubric_grading_items,
         {
           manual_rubric_grading_id: example.manual_rubric_grading_id,
         },
         RubricItemSchema,
       );
-      // Warning when graded rubric item does not match current rubric item
-      if (
-        rubric_grading_items.length > 0 &&
-        rubric_grading_items[0].rubric_id !== rubric_items[0].rubric_id
-      ) {
-        warning += `Instance question ${example.instance_question_id}: example rubric id is different from the current rubric id.\n`;
-      }
```

Comment:
As suggested by @mwest1066 (https://github.com/PrairieLearn/PrairieLearn/pull/10726#discussion_r1964660125), it doesn't necessarily make sense to warn in this case. At any rate, it isn't actionable.